sdks/java/core/src/main/java/org/apache/beam/sdk/util/GcsUtil.java

```diff
@@ -526,7 +532,7 @@ public Void call() throws IOException {
     List<BatchRequest> batches = new LinkedList<>();
     for (List<GcsPath> filesToGet :
         Lists.partition(Lists.newArrayList(paths), MAX_REQUESTS_PER_BATCH)) {
-      BatchRequest batch = storageClient.batch();
+      BatchRequest batch = storageClient.batch(httpRequestInitializer);
```

Comment:
Is there a way to refactor this code slightly so that the code to create a `BatchRequest` only needs to exist and be right in 1 place, instead of in 4 places?

Otherwise, need to test all 4 places.